src/core/validations/client_parity_tests/generate-messages.ts

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env ts-node
```

Comment:
We should put this into a separate directory that's outside of `src` so we don't mix rust and typescript code. Maybe `scripts/validations`?